src/sentry/tasks/post_process.py

```diff
@@ -1512,6 +1513,11 @@ def detect_new_escalation(job: PostProcessJob):
         return
 
 
+def detect_hostnames_for_uptime(job: PostProcessJob):
+    url = get_path(job.event.data, "request", "url")
+    detect_hostname_for_project(job.event.project, url)
```

Comment:
This is pretty fast so I haven't made it a task, but we could easily split it out into a task since we just need the url and project id so we can fetch the project.